dummy_app.py

```diff
@@ -1,20 +1,28 @@
-#from tempfile import NamedTemporaryFile
 import streamlit as st
 from ms2query.utils import json_loader
 
+st.title("Query a Spec2Vec model and plot the results!")
 st.write("""
-# Dummy app to get started
-First goal is simply to plot a spectrum using **matchms**!
+Upload your query and library spectra files in json format.
+Query the library using a Spec2Vec model and inspect the results! 
 """)
 
-uploaded_file = st.file_uploader("Choose a spectrum file...", type=['json', 'txt'])
-#temp_file = NamedTemporaryFile(delete=False)
-if uploaded_file is not None:
-    #temp_file.write(uploaded_file.getvalue())
-    if uploaded_file.name.endswith("json"):
-        #spectrums = json.load(uploaded_file) 
-        spectrums = json_loader(uploaded_file)
-        st.write(spectrums[0].metadata)
-        
-        fig = spectrums[0].plot()
-        st.pyplot(fig)
\ No newline at end of file
+# load query file in sidebar
+query_file = st.sidebar.file_uploader("Choose a query spectrum file...",
```

Comment:
Nice to have it in the sidebar!
How complicated would it be to add a button to it like ``load example`` or similar?

Reply:
Same actually also for the library import.

Reply:
Yes I was thinking the same! I don't think it would be too hard.. I have seen some button things in the docs. I will have a look :